api-interop-layer/data/alerts/backgroundUpdateTask.js

```diff
@@ -214,9 +225,12 @@ export const start = () => {
 };
 
 if (parentPort) {
-  parentPort.on("message", (message) => {
-    switch (message.toLowerCase()) {
+  parentPort.on("message", ({ action, data }) => {
+    switch (action.toLowerCase()) {
       case "start":
+        data.forEach((hash) => {
+          KNOWN_ALERTS.add(hash);
+        });
```

Comment:
When the background process starts up, it can now take a list of known alert hashes as seed. This is useful for cases where the background process needs to be restarted. The main process can send it the list of hashes it knows about and then the background process can start off skipping those.